src/topology/sheaves/sheaf_condition/pairwise_intersections.lean

```diff
@@ -0,0 +1,334 @@
+      { ext, dsimp, simp, },
+      { ext ⟨i,j⟩, dsimp, simp, },
+    end, }}) (by tidy)
+
+/--
+Cones over `diagram U ⋙ F` are the same as a cones over the usual sheaf condition equalizer diagram.
+-/
+@[simps]
+def cone_equiv (F : presheaf C X) ⦃ι : Type v⦄ (U : ι → opens X) :
+  limits.cone (diagram U ⋙ F) ≌ limits.cone (sheaf_condition_equalizer_products.diagram F U) :=
+{ functor := cone_equiv_functor F U,
+  inverse := cone_equiv_inverse F U,
+  unit_iso := cone_equiv_unit_iso F U,
+  counit_iso := cone_equiv_counit_iso F U, }
+
+local attribute [reducible]
+  sheaf_condition_equalizer_products.res
+  sheaf_condition_equalizer_products.left_res
+
+/--
+If `sheaf_condition_equalizer_products.fork` is an equalizer,
+then `F.map_cone (cone U)` is a limit cone.
+-/
+def is_limit_map_cone_of_is_limit_sheaf_condition_fork
+  (F : presheaf C X) ⦃ι : Type v⦄ (U : ι → opens X)
+  (P : is_limit (sheaf_condition_equalizer_products.fork F U)) :
+  is_limit (F.map_cone (cone U)) :=
+is_limit.of_iso_limit ((is_limit.of_cone_equiv (cone_equiv F U).symm).symm P)
+{ hom :=
+  { hom := 𝟙 _,
+    w' :=
+    begin
+      rintro ⟨⟩,
+      { dsimp, simp, refl, },
+      { dsimp,
+        simp only [limit.lift_π, limit.lift_π_assoc, category.id_comp, fan.mk_π_app, category.assoc],
+        rw ←F.map_comp,
+        refl, }
+    end },
+  inv :=
+  { hom := 𝟙 _,
+    w' :=
+    begin
+      rintro ⟨⟩,
+      { dsimp, simp, refl, },
+      { dsimp,
+        simp only [limit.lift_π, limit.lift_π_assoc, category.id_comp, fan.mk_π_app, category.assoc],
+        rw ←F.map_comp,
+        refl, }
+    end }, }
+
+/--
+If `F.map_cone (cone U)` is a limit cone,
+then `sheaf_condition_equalizer_products.fork` is an equalizer.
+-/
+def is_limit_sheaf_condition_fork_of_is_limit_map_cone
+  (F : presheaf C X) ⦃ι : Type v⦄ (U : ι → opens X)
+  (Q : is_limit (F.map_cone (cone U))) :
+  is_limit (sheaf_condition_equalizer_products.fork F U) :=
+is_limit.of_iso_limit ((is_limit.of_cone_equiv (cone_equiv F U)).symm Q)
+{ hom :=
+  { hom := 𝟙 _,
+    w' :=
+    begin
+      rintro ⟨⟩,
+      { dsimp, simp, refl, },
+      { dsimp, ext ⟨i, j⟩,
+        simp only [limit.lift_π, limit.lift_π_assoc, category.id_comp, fan.mk_π_app, category.assoc],
+        rw ←F.map_comp,
+        refl, }
+    end },
+  inv :=
+  { hom := 𝟙 _,
+    w' :=
+    begin
+      rintro ⟨⟩,
+      { dsimp, simp, refl, },
+      { dsimp, ext ⟨i, j⟩,
+        simp only [limit.lift_π, limit.lift_π_assoc, category.id_comp, fan.mk_π_app, category.assoc],
+        rw ←F.map_comp,
+        refl, }
+    end }, }
+
+
+end sheaf_condition_pairwise_intersections
+
+open sheaf_condition_pairwise_intersections
+
+/--
+The sheaf condition in terms of an equalizer diagram is equivalent
+to the reformulation in terms of a limit diagram over `U i` and `U i ⊓ U j`.
+-/
+def sheaf_condition_equiv_sheaf_condition_pairwise_intersections (F : presheaf C X) :
+  F.sheaf_condition ≃ F.sheaf_condition_pairwise_intersections :=
```

Comment:
Now I'm confused... is there still data left in `F.sheaf_condition` after the `prop-limits` merge?

Reply:
Yes. `sheaf_condition` contains `is_limit`, which has always been a subsingleton (but nevertheless data).

Reply:
Do we want to turn that into a Prop as well, some day?